sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3748,7 +3748,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
 
   test("SPARK-33084: Add jar support Ivy URI in SQL") {
     val sc = spark.sparkContext
-    val hiveVersion = "2.3.9"
+    val hiveVersion = "2.3.9" // TODO update to 2.3.10 after jar available in maven central
```

Comment:
I'm not sure if this TODO is really required when we have a staging repository, @pan3793 .

To be sure, we need to pass all the tests.

Reply:
this one is actually irrelevant, see SPARK-47928(https://github.com/apache/spark/pull/46150)